django/contrib/auth/backends.py

```diff
@@ -90,9 +96,15 @@ def has_module_perms(self, user_obj, app_label):
     def get_user(self, user_id):
         UserModel = get_user_model()
         try:
-            return UserModel._default_manager.get(pk=user_id)
+            user = UserModel._default_manager.get(pk=user_id)
         except UserModel.DoesNotExist:
             return None
+        if user:
+            is_active = getattr(user, 'is_active', None)
+            if (is_active or (is_active is False and self.allow_inactive_users) or
```

Comment:
Isn't `is_active is False` redundant? We already check if it's truthy in the first condition so if we get here it must be falsey.